.bingo/yq.mod

```diff
@@ -0,0 +1,5 @@
+module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
+
+go 1.19
```

Comment:
Odd... I'm also using 4.34.1 in downstream operator-controller, but it uses go1.20 for me.  Perhaps the env where you added this was using 1.19?

Reply:
I see it inconsistently 1.19/1.20 in other bingo-managed bins, so it's harder for me to figure out the origin.  If you can, please update everything to standardize on 1.20.

Reply:
Addressing here: https://github.com/operator-framework/rukpak/pull/648